content/webapp/services/prismic/types/images.ts

```diff
@@ -9,7 +9,7 @@ type Params = {
 };
 
 export function getImageUrlAtSize(
-  image: Image | Picture,
+  image: Image | Omit<Image, '32:15' | '16:9' | 'square'> | Picture,
```

Comment:
Might be worth adding a comment as to why we're omitting these crops?

Reply:
I actually don't think we need to do this, I'll remove it. Think I put it in temporarily as it was being noisy during the process of doing this work. Thanks for catching